docs/documentation/usage.rst

```diff
@@ -316,6 +285,112 @@ Several constraint variables can be defined.
 
 Also, see :ref:`get-variable-list`.
 
+***************************
+Advanced configuration file
+***************************
+
+Sys path
+========
+.. code:: yaml
+
+    sys_path:
+      - /path/to/your/module
+
+This section is used to add a path to the Python `sys.path`.
+This is useful when you have a module that is not in the Python path.
+Note that this setting affects only the `imports` section below, which is used for setting drivers and solvers.
+
+
+Imports
+=======
+.. code:: yaml
+
+    imports:
+        my_driver_1: MyDriver1
+        utils.my_drivers.my_driver_2: MyDriver2
+        utils.my_solvers.my_favourite_solver: MyFavouriteSolver
+
+This section is used to import modules such as solvers and drivers that are not available in OpenMDAO.
+The key is the path of the module or Python file, and the value is the name of the object to import.
+The YAML code lines above will do the equivalent in Python of:
+
+.. code:: python
+
+    from my_driver_1 import MyDriver1
+    from utils.my_drivers.my_driver_2 import MyDriver2
+    from utils.my_solvers.my_favourite_solver import MyFavouriteSolver
+
+
+Driver settings
+===============
+
+The `driver` configuration can be specified in two ways: using the standard syntax (a string) or the advanced syntax (an object).
+
+Standard Syntax
+---------------
+
+In the standard syntax, the driver is specified as a string. This string should be a valid Python expression that creates an instance of an OpenMDAO driver.
+
+.. code:: yaml
+    driver: om.ScipyOptimizeDriver(tol=1e-2, optimizer='COBYLA')
```

Comment:
Ok, but still not ok. This part does not appear in the rendered doc (I know, ReST sucks...)
```suggestion
.. code:: yaml

    driver: om.ScipyOptimizeDriver(tol=1e-2, optimizer='COBYLA')
```